internal/terraform/module/module_loader_test.go

```diff
@@ -64,6 +64,11 @@ func TestModuleLoader_referenceCollection(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	varsOriginsOp := NewModuleOperation(modPath, op.OpTypeDecodeVarsReferences)
+	err = ml.EnqueueModuleOp(varsOriginsOp)
```

Comment:
I _think_ this part of the test isn't going to be very useful unless we also add tfvars file(s) to the test module in `testdata/single-root-no-modules`. It may be fine just to leave this test as it was though. The logic seems well tested in isolation.